easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -79,6 +79,8 @@
 # set of configure/build/install options that can be provided as lists for an iterated build
 ITERATE_OPTIONS = ['preconfigopts', 'configopts', 'prebuildopts', 'buildopts', 'preinstallopts', 'installopts']
 
+PRESERVE_TEMPLATE_REGEX = re.compile(r'(%)(?!%*\(\w+\)s)')
```

Comment:
just make the actual regex a global constant? There is no benefit from storing the compiled version.